packages/x-components/tests/e2e/cucumber/tagging/tagging.spec.ts

```diff
@@ -29,15 +35,15 @@ Then('result click tagging request is triggered', () => {
 });
 
 Then('query tagging request should be triggered', () => {
-  cy.wait('@queryTagging');
+  cy.wait('@queryTagging').should('exist');
 });
 
 Then('query tagging request has been triggered', () => {
-  cy.get('@queryTagging').should('exist');
+  cy.get('@queryTagging', { timeout: 0 }).should('exist');
```

Comment:
Sadly, I'm afraid to say this is causing some tests to fail locally. And they keep failing randomly (sometimes they pass sometimes they don't 🤷🏿‍♂️ ).

![image](https://user-images.githubusercontent.com/6247440/154278554-3867d022-a287-448a-bfba-36dc78644d60.png)

Both tests failing in this image is due to same error in this line.

Reply:
As we talked in private, this seems to be working as expected (something might have been wrong in my local environment by the time I ran the tests).